go.mod

```diff
@@ -4,11 +4,11 @@ go 1.14
 
 require (
 	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.1
```

Comment:
While running `make test`, this line is regenerated again, so any pointers how to remove a module from the package itself?

Reply:
This won't disappear, the new library imports the old one still. This is fine.